literature-cleaned.bib

```diff
@@ -5261,6 +5271,15 @@ @article{AGA:TSE13
 	year = 2013
 }
 
+@inproceedings{CGSS:TACAS13,
+  title= {{The mathsat5 smt solver}},
+  author={Cimatti, Alessandro and Griggio, Alberto and Schaafsma, Bastiaan Joost and Sebastiani, Roberto},
+  booktitle = TACAS,
+  pages={93--107},
+  year= 2013,
+  organization={Springer}

```

Comment:
Change `organization` to `publisher` and remove brackets? `publisher` and `address` are missing. They should be always part of inproceedings.